WeatherStationClient.py

```diff
@@ -18,6 +41,31 @@ def __init__(self, code, response):
         # Better safe than sorry? If the server returns a non-empty response with code >= 300 we should preserve it
         self.response = response
 
+    @classmethod
+    def _get_specific(cls, code, response):
```

Comment:
I would just make dictionary.

Reply:
Nie jestem pewny, czy rozumiem, o co Ci chodzi.

Czy o coś takiego?

    specific_types = {
        400: ApiResponseExceptionBadRequest,
        401: ApiResponseExceptionUnauthorized,
        403: ApiResponseExceptionForbidden,
        409: ApiResponseExceptionConflict
    }
    return specific_types[code] if code in specific_types else cls
else cls